solr/CHANGES.txt

```diff
@@ -96,6 +96,8 @@ Improvements
 
 * SOLR-17041: Make CommitTracker currentTlogSize lazy (Alex Deparvu)
 
+* SOLR-17050: Use compact JSON for Learning to Rank (LTR) feature and model storage. (Florin Babes, Christine Poerschke)
+
```

Comment:
https://github.com/apache/solr/pull/2030/commits/e369a6c963234f123a81a9c61bfbbfeb6530a97d undoes the CHANGES.txt entry addition temporarily, to be added back before merging.

Reply:
@cpoerschke  I have to add the entry in CHANGES.txt ?

Reply:
> @cpoerschke I have to add the entry in CHANGES.txt ?

Thanks for asking! I've added the entry back in https://github.com/apache/solr/pull/2030/commits/02d166fe06763164289cff59413ee32c7a1943c1 after merging in the latest `origin/main` -- CHANGES.txt more easily encounters merge conflicts and sometimes it is easier to defer the entry addition until closer to merge time.